conf/bkenv.sh

```diff
@@ -18,6 +18,8 @@
 # under the License.
 #
 
+# NOTE: this script is intentionally not executable. It is only meant to be sourced for environment variables.
```

Comment:
what about removing the execution permission ? in this way you can run `source conf/bkenv.sh` but you can't run it using `./conf/bkenv.sh`

btw not sure it's worth to do that 

Reply:
The `conf/bkenv.sh` file does not currently have execute permissions. It is used in a few places with commands like:

```bash
source $BK_HOME/conf/bkenv.sh
```

and 

```bash
# Check bookkeeper env and load bkenv.sh
if [ -f "$PULSAR_HOME/conf/bkenv.sh" ]
then
    . "$PULSAR_HOME/conf/bkenv.sh"
fi
```

I added this note because it surprised me that a `.sh` file was in the `conf` directory. If the note is obvious/confusing, I can remove it.